core-tests/src/test/java/org/mule/runtime/core/api/MessagingExceptionTestCase.java

```diff
@@ -345,7 +344,7 @@ public void payloadInfoNonConsumable() throws Exception {
     when(payload.toString()).then(new FailAnswer("toString() expected not to be called."));
     Message muleMessage = of(payload);
 
-    when(transformationService.transform(muleMessage, DataType.STRING)).thenReturn(of(value));
+    when(transformationService.internalTransform(muleMessage, DataType.STRING)).thenReturn(of(value));
```

Comment:
why stub this method instead of transform? This comment applies to this and all the uses below